### ISSUE TITLE
False negatives in NullableToStringCall

### ISSUE THREAD

**Issue by t-kameyama** [detekt/detekt#3196]
```kotlin
data class Foo(val a: Any?)

fun test(foo: Foo) {
    val x = foo.a.toString()
    val y = "${foo.a}"
}
```

**Comment by t-kameyama** [comment]
I'm working on this issue.
